### PR TITLE
Selenium WebDriver Basics, Protractor + Locators

### PR DESCRIPTION
**Pull request by alex39292**

### CODE REVIEW

File: test/module_6/protractorTask.spec.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+const { browser, element, $ } = require("protractor");
```
Comment (SiarheiRylach) [inline test/module_6/protractorTask.spec.js]: not necessary

File: test/module_6/protractorTask.spec.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+describe("Protractor Demo App", () => {
+    const firstNumber = element(by.model("first"));
+    const secondNumber = element(by.model("second"));
+    const goButton = element(by.id("gobutton"));
+    const operator = element(by.model("operator"));
+    const history = element.all(by.css("tr.ng-scope"));
+
+    browser.get("http://juliemr.github.io/protractor-demo/");
```
Comment (SiarheiRylach) [inline test/module_6/protractorTask.spec.js:8]: beforeEach?

Comment (SiarheiRylach) [inline test/module_6/protractorTask.spec.js:8]: for future: I would recommend using async/await or promise chaining for async functions. I wouldn't hope on control flow.

File: test/module_6/webDriverJS.spec.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+const { describe, afterEach, it } = require("mocha");
+const { expect } = require("chai");
+const {Builder, By, until} = require("selenium-webdriver");
+const driver = new Builder().forBrowser("firefox").build();
+
+describe("Selenium-webdriverJS test", function () {
+  afterEach(async () => await driver.quit());
+
+  it("oz.by simple test: find a book, add to cart, delete from cart", async () => {
+      await driver.get("https://oz.by/")
```
Comment (SiarheiRylach) [inline test/module_6/webDriverJS.spec.js:10]: fyi: you can use `return` instead of `await`

File: test/module_6/protractorTask.spec.js (hunk 1 above)
Comment (alex39292) [reply test/module_6/protractorTask.spec.js]: deleted it

File: test/module_6/protractorTask.spec.js (hunk 2 above)
Comment (alex39292) [reply test/module_6/protractorTask.spec.js:8]: added beforeEach()

Comment (alex39292) [reply test/module_6/protractorTask.spec.js:8]: ok)

File: test/module_6/webDriverJS.spec.js (hunk 3 above)
Comment (alex39292) [reply test/module_6/webDriverJS.spec.js:10]: thanks

**Review by SiarheiRylach — APPROVED** [review]

**Pull request merged**